### DISCUSSION TITLE
E21 Non modifiable

### DISCUSSION THREAD

**Discussion by naidaon** [mason-org/mason.nvim#1574] [Q&A]
Hi, I was hoping someone could tell me why when I try to delete LSPs so I can reinstall them it comes up as 'E21: Cannot make changes, 'modifiable' is off' I have tries `:setglobal modifiable` and :set modifiable` but with no sucess

**Comment by OWL4C** [comment]
I have a similar error and using a Capital X (Shift+X) works. Using :MasonUninstall still works too

**Comment by williamboman** [comment]
Are you sure you're not pressing <kbd>x</kbd> (lowercase) instead of `X` (<kbd>shift</kbd> + <kbd>x</kbd>)?
